lib/debug/session.rb

```diff
@@ -231,25 +231,36 @@ def wait_command
       when 'c', 'continue'
         @tc << :continue
 
-      # * `q[uit]` or exit or `Ctrl-D`
+      # * `q[uit]` or `Ctrl-D`
       #   * Finish debugger (with the debuggee process on non-remote debugging).
-      when 'q', 'quit', 'exit'
+      when 'q', 'quit'
         if ask 'Really quit?'
           @ui.quit arg.to_i
           @tc << :continue
         else
           return :retry
         end
 
-      # * `kill` or `q[uit]!`
-      #   * Stop the debuggee process.
-      when 'kill', 'quit!', 'q!'
+      # * `q[uit]!`
+      #   * Same as q[uit] but without the confirmation prompt.
+      when 'q!', 'quit!'
+        @ui.quit arg.to_i
+        @tc << :continue
+
+      # * `kill`
+      #   * Stop the debuggee process with `Kernal#exit!`.
```

Comment:
@ko1 I think it's `Kernel#exit!`, right?

I can't speak for other people, but as a Rubyist with 8 years of experience, I don't remember I've ever used the `exit!` method in my codebase. I think for people who use Ruby for scripting they should be familiar with it. But Rails or web-focused Ruby developers may not be familiar with this method either. 

TBH, when I saw `exit` the first thing that came into my mind was the `exit` command in `pry` or `irb`. That's why I initially proposed `exit!` in the issue 😂

Reply:
BTW, maybe pry and irb's `exit` command is only eval "exit" string and calls `Kernel#exit` method by eval.

Reply:
In `pry`, there's an [exit command class](https://github.com/pry/pry/blob/0aae8c94ad03a732659ed56dcd5088469a15eebf/lib/pry/commands/exit.rb), which I always thought is the source of the `exit` command. But when I check the source of `exit` inside the repl, I can only see 

```
> show-source exit

From: process.c (C Method):
Owner: Kernel
Visibility: private
Signature: exit(*arg1)
Number of lines: 5

static VALUE
f_exit(int c, const VALUE *a, VALUE _)
{
    return rb_f_exit(c, a);
}
```

This is mind blowing 🤯

Reply:
Ah, my mistake :p